src/js/common/components/Modal.jsx

```diff
@@ -26,9 +26,11 @@ class Modal extends React.Component {
   componentWillReceiveProps(newProps) {
     if (newProps.visible && !this.props.visible) {
       this.setState({ lastFocus: document.activeElement, focusListener: focusListener(newProps.focusSelector) });
+      document.body.classList.add('modal-open');
```

Comment:
I wonder if these should be in a different lifecycle method. Generally I've seen that people keep DOM operations in componentWillUpdate/componentDidUpdate/componentDidMount (though I bent that rule for the event listener code).

This method also doesn't run when a component is initially rendered, so if you render a page that has a modal open to start with, I don't think `modal-open` will be added to the body.